src/plugins/intel_cpu/tests/unit/ngraph_transformations/convert_matmul_test.cpp

```diff
@@ -412,17 +289,14 @@ TEST_F(TransformationTestsF, ConvertMatMulToFCTest_second_input_rank_adj_3) {
 
         function = std::make_shared<ngraph::Function>(ngraph::NodeVector{ add }, ngraph::ParameterVector{ input1 });
         manager.register_pass<ConvertMatMulToFC>();
-        manager.register_pass<FullyConnectedBiasFusion>();
     }
     {
         auto input1 = std::make_shared<ngraph::opset1::Parameter>(ngraph::element::f32, ngraph::Shape{ 5, 2, 3 });
-        auto reshape_before_const = ngraph::opset1::Constant::create(ngraph::element::i64, { 2 }, { -1, 3 });
 
         auto weights = ngraph::opset1::Constant::create(ngraph::element::f32, ngraph::Shape{ 2, 3 }, { 1 });
-        auto biases = ngraph::opset1::Constant::create(ngraph::element::f32, ngraph::Shape{ 2 }, { 1 });
```

Comment:
Can I ask you to add one more test case with 3D input but without bias (by analogy with `ConvertMatMulToFCTest_second_input_rank_adj_2`)?